src/wrapping.rs

```diff
@@ -63,6 +70,70 @@ impl<T: ConstantTimeEq> ConstantTimeEq for Wrapping<T> {
     }
 }
 
+#[cfg(feature = "num-traits")]
+impl<T: Zero> num_traits::Zero for Wrapping<T>
```

Comment:
Seems like this (and several of the other impls) can just be blanket impls like:

```
impl<T: Zero> num_traits::Zero for T
```

...size `Wrapping<T>` already impls `Zero` using this same pattern

Reply:
Never mind, I suppose such a blanket impl wouldn't be permitted.